pkgs/top-level/ocaml-packages.nix

```
@@ -572,6 +572,8 @@ let
 
     lua-ml = callPackage ../development/ocaml-modules/lua-ml { };
 
+    luv = callPackage ../development/ocaml-modules/luv { };
```

Make sure `file` comes from top-level, it's easily imaginable we'll have a ocaml package named `file` one day.

```suggestion
    luv = callPackage ../development/ocaml-modules/luv {
      inherit (pkgs) file;
    };
```